tests/infrastructure/golden_images/update_boot_source/test_latest_rhel_image.py

```diff
@@ -85,6 +85,7 @@ def rhel_vm_minor_ver_num(rhel_vm):
     ],
     indirect=True,
 )
+@pytest.mark.s390x
```

Comment:
I think you can remove this, it has no effect on the cluster type so running it on any cluster is ok, it tests that the image we use from the data import cron is updated to the latest version